cypress/integration/tests/controls/jobfunctions/filter.test.ts

```diff
@@ -0,0 +1,59 @@
+/// <reference types="cypress" />
+
+import { login, clickByText, exists, applySearchFilter } from "../../../../utils/utils";
+import { navMenu, navTab } from "../../../views/menu.view";
+import {
+    controls,
+    jobfunctions,
+    button,
+    nameFilter,
+    clearAllFilters,
+} from "../../../types/constants";
+
+import { Jobfunctions } from "../../../models/jobfunctions";
+import * as data from "../../../../utils/data_utils";
+
+var jobfunctionsList: Array<Jobfunctions> = [];
+var invalidSearchInput = data.getRandomWord(5);
```

Comment:
I tried this, but sometimes it returns a word that exists in already created job functions. 

Example - Test created a job function named - **_"Interaction"_**, and then this random word of 3 chars returned **_"int"_**. And hence, it was a valid search string, causing test to fail. Happens very rarely, but we can go with totally non-searchable fixed string like - "uuuuu" or "1111" etc.